Makefile.am

```diff
@@ -54,10 +54,10 @@ tests_gds_kernel_loopback_latency_LDADD = $(top_builddir)/src/libgdsync.la $(LIB
 SUFFIXES= .cu
 
 .cu.o:
-	$(NVCC) $(CPPFLAGS) $(NVCCFLAGS)  -c -o $@ $<
+	$(NVCC) $(CPPFLAGS) $(AM_CPPFLAGS) $(NVCCFLAGS)  -c -o $@ $<
```

Comment:
this change does not belong to this branch. you should rebase this branch from devel